### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by MMhaidari**
# What is added new

- [ ] RSpec added as a gem to project.
- [ ] Spec file for all your controllers created.
- [ ] Check for Below action:
- If response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        get '/users'
+      end
+
+      it 'success for index action' do
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'Render correct templete for index action' do
+        expect(response).to render_template(:index)
+      end
```
Comment (tresorsawasawa) [inline spec/controllers/users_controller_spec.rb]: - [ ] As required for the project, kindly ensure that your index specs include tests that ensures the response body includes the correct placeholder text. Example : `expect(response.body).to include("Welcome to user index page")`

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```
Comment (tresorsawasawa) [inline spec/controllers/users_controller_spec.rb]: - [ ] Also for best practices and as required for the project, kindly ensure that both your `users_controller_spec.rb` and `posts_controller_spec.rb` spec files are in the `spec/requests/` directory. This means you should edit/change this directory name `spec/controllers/` to ``spec/requests/``

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @MMhaidari 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- All your tests are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MMhaidari** [comment]
# What is added new 
- [ ] requested changes have added as instructed

**Review by Gambit142 — APPROVED** [review]
Hi @MMhaidari 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linter checks passed ✔️ 
- Installed `rspec` gem ✔️ 
- Wrote requests tests for `Users` controllers ✔️ 
- Wrote requests tests for `Posts` controllers ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**